site/en/blog/third-party-scripts/index.md

```diff
@@ -0,0 +1,264 @@
+
+{% Img src="image/1L2RBhCLSnXjCnSlevaDjy3vba73/U26McnxB3ZYKJ2zE0Zp4.png", alt="Third Party Web visualization", width="800", height="500" %}
+
+"Across top ~4 million sites, ~2700 origins account for ~57% of all script execution time with the top 50 entities already accounting for ~47%". - [third-party-web](https://github.com/patrickhulce/third-party-web)
+
+Pages that render quickly and become interactive within a reasonable timeframe are essential to a good user experience as measured by the CWV. Good UX often translates to good business for websites, which can mean good business for third-parties being used. Working together to reduce the impact of third-parties can be a win for everyone in the chain.
+
+We acknowledge that Google vends a number of commonly used third-party scripts, including Google Tag Manager, YouTube embeds, and ReCaptcha to name a few. We acknowledge that a number of our scripts could have a lighter performance impact on the Core Web Vitals, and we are committed to exploring ways to improve this impact where possible.
+
+# How can Chrome help?
+
+Third-party resources being poor for performance is often a tragedy of the commons requiring a step-change in the underlying ecosystem dynamics. Chrome wants to drive innovation to achieve the following outcomes:
+
+1.  ***Find better ways of loading third-party resources on the web while not regressing user-experience or business outcomes.***
+
+    We know that we cannot proceed far in this effort if we don't collaborate with partners, businesses, third-parties, and developers. We want to create an open field to discuss the possibilities and exchange ideas via public explainers and specifications. Developers will have time to provide feedback and test the impact of many of these proposals.
+
+2.  ***Enable users of third-party scripts to have better attribution for their costs in tooling and in the field, clear, well-paved paths for their usage, and better incentives during authoring time to ensure they are optimal by default.***
+
+    We want to enhance all layers, such as user agents, frameworks, and third-party scripts to reduce the performance impact of third-parties. We intend to also provide sufficient insights to help site owners employ best practices around each script embedded, including faster alternatives where applicable.
+
+## Our Approach
+
+We propose a three-pronged approach for achieving these outcomes..
+
+1.  **Give ***developers*** deeper attribution into per third-party impact via RUM and in Google's developer tooling.**
```

Comment:
Just me, but did you check if this markdown syntax parses? There's something about it that seems off. No biggie if it parses, but wanted to avoid any rendering anomalies in the final post.